argostime/crawler/crawl_utils.py

```diff
@@ -22,13 +23,42 @@
     along with Argostimè. If not, see <https://www.gnu.org/licenses/>.
 """
 
+import configparser
 import logging
 import re
-from typing import Optional
+from typing import Callable, Optional
 
-voor_regex = re.compile("voor")
+__config = configparser.ConfigParser()
+__config.read("argostime.conf")
+__voor_regex = re.compile("voor")
 
-class CrawlResult():
+enabled_shops = {}
```

Comment:
This is missing a type annotation

Reply:
> Ziet er goed uit, heb het zelf nog niet gedraaid maar als het bij jou werkt zal het wel prima zijn.

Famous last words...

> Wat me wel een beetje onduidelijk is, is wanneer `enabled_shops` nou daadwerkelijk gevuld wordt - gewoon eenmalig op startup neem ik aan? Kan niet zo goed een duidelijke call vinden die dat doet

Ja, eenmalig op startup. Iets precieser als je deze import doet:
```
from argostime.crawler.shop import *
```
dan import je `__all__` wat gedefineerd is in `argostime.crawler.shop.__init__.py`. En als je deze import doet worden de decorators `@register_crawler(name: str, host: str)` aangeroepen. Die decorator is in `argostime.crawler.crawl_utils.py` gedefineerd en maakt de entry in `enabled_shops` aan (tenzij de shop disabled is).